packages/app/pages/index.tsx

```diff
@@ -11,11 +11,19 @@ import React, {
   useState,
 } from "react";
 import Web3 from "web3";
+import { Contract } from "web3-eth-contract";
+import { TransactionReceipt } from "web3-core";
 import { AbiType, StateMutabilityType } from "web3-utils";
 import { Modal, Input, Button } from "@cabindao/topo";
 import { styled } from "../stitches.config";
+import BN from "bn.js";
 import passportFactoryJson from "../../contracts/artifacts/contracts/PassportFactory.sol/PassportFactory.json";
 
+const contractAddressesByNetworkId: {
+  [id: number]: { passportFactory: string };
+} = {
+  0x7a69: { passportFactory: "0x5FbDB2315678afecb367f032d93F642f64180aa3" },
```

Comment:
looks like the contract consistently deploys to this same address locally, at least for me. wonder as more people onboard if this is consistent.

We'll need to add more contract addresses here as we deploy it to test/main nets. wonder if there's a better way to manage this or if hardcoded is fine